describe/describe.py

```diff
@@ -38,97 +38,103 @@ def summarize(
     :param plot_hist:       (True) set this to False for large tables
     :param plots_dest:      destination folder of summary plots (relative to artifact_path)
     :param update_dataset:  when the table is a registered dataset update the charts in-place
+    :param frac:  when the table has more than 5000 samples, the function will execute on random frac from the data
     """
+
     df = table.as_df()
-    header = df.columns.values
+    if df.shape[0] > 5000:
+        df = df.sample(frac=frac)
     extra_data = {}
 
+    # histograms
```

Comment:
Please split each plot to another function that receives the required data and configurations (will now have default values for the configurations, later we will want to enable edit them).

Later each function will be converted into a `DatasetPlan` like we talked about at the office.